dumper-integration-tests/redshift-tests/src/main/java/com/google/edwmigration/dumper/jdbc/JdbcUtil.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.jdbc;
+
+import static com.google.edwmigration.dumper.base.TestConstants.TRAILING_SPACES_REGEX;
+
+import java.math.BigDecimal;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.ZonedDateTime;
+import java.util.Calendar;
+import java.util.TimeZone;
+
+/**
+ * A helper class for checking Null values returned by executing SELECT request against a database.
+ */
+public final class JdbcUtil {
+
+  private JdbcUtil() {}
+
+  /**
+   * @param rs A row with SELECT results.
+   * @param column Database column name.
+   * @return String or an empty string if null.
+   */
+  public static String getStringNotNull(ResultSet rs, String column) throws SQLException {
+    String string = rs.getString(column);
+    return rs.wasNull() ? "" : TRAILING_SPACES_REGEX.matcher(string).replaceFirst("");
+  }
+
+  /**
+   * @param rs A row with SELECT results.
+   * @param column Database column name.
+   * @return int or 0 if null.
+   */
+  public static int getIntNotNull(ResultSet rs, String column) throws SQLException {
+    return rs.getInt(column);
+  }
+
+  /**
+   * @param rs A row with SELECT results.
+   * @param column Database column name.
+   * @return long or 0L if null.
+   */
+  public static long getLongNotNull(ResultSet rs, String column) throws SQLException {
+    return rs.getLong(column);
+  }
+
+  /**
+   * @param rs A row with SELECT results.
+   * @param column Database column name.
+   * @return byte[] or empty byte[] if null.
+   */
+  public static byte[] getBytesNotNull(ResultSet rs, String column) throws SQLException {
+    try {
+      byte[] bytesValue = rs.getBytes(column);
+      return rs.wasNull() ? new byte[0] : bytesValue;
+    } catch (SQLException e) {
+      BigDecimal bigDecimal = rs.getBigDecimal(column);
+      return rs.wasNull() ? new byte[0] : bigDecimal.toBigInteger().toByteArray();
```

Comment:
Why/when does this case happen, please?

Reply:
It happened with Teradata DB. 
This code came here because we are using pretty much the same test automation framework skeleton and its util methods.
I'll remove it entirely.

Done.